ragas-eval-flow/README.md

```diff
@@ -0,0 +1,92 @@
+# Ragas Evaluation Flow
+
+This evaluation flow uses the [ragas evaluation framework](https://github.com/explodinggradients/ragas) to calculate evaluation metrics.
+
+> *Ragas is a framework that helps you evaluate your Retrieval Augmented Generation (RAG) pipelines. Ragas provides you with the tools based on the latest research for evaluating LLM-generated text to give you insights about your RAG pipeline. Ragas can be integrated with your CI/CD to provide continuous checks to ensure performance.*
+
+See [Ragas documentation](https://docs.ragas.io/en/latest/) for more information.
+
+## Inputs
+
+- **question**: original user prompt
+- **answer**: answer generated by the model
+- **context**: context provided to the model in a prompt
+- **ground_truth**: expected answer
+
+## Output
+
+A batch run of the flow creates an output that contains a JSON object of the following shape **for each input in the
+data set**:
+
+```json
```

Comment:
nit: add short explanation about the values
I know these are in the official docs, but it would be nice to have them here as a quick reference.
For example:
```
faithfulness - the factual consistency of the answer to the context base on the question.
context_precision - a measure of how relevant the retrieved context is to the question. Conveys quality of the retrieval pipeline.
answer_relevancy - a measure of how relevant the answer is to the question
context_recall: measures the ability of the retriever to retrieve all the necessary information needed to answer the question.
```